src/components/EditableSelectList.vue

```diff
@@ -114,7 +114,7 @@ export default {
   }
 }
 .editable-table {
-  overflow-x: scroll;
+  overflow-x: auto;
```

Comment:
Do we need this in both components?

Reply:
Oh, probably not. I'll try and remove it.